### DISCUSSION TITLE
Support for JetBrains IDE

### DISCUSSION THREAD

**Discussion by Damecek** [forcedotcom/Einstein-GPT-for-Developers#22] [Ideas]
Hi, thanks for the VS Code plugin. Could you please consider also support for JetBrains IDE like Idea?

**Comment by Damecek** [comment]
Looks like IC2 is already implementing support for Einstein GPT. https://illuminatedcloud.blogspot.com/2023/09/2283-release-notes.html

**Comment by klewis-sfdc** [comment]
I converted this Issue to a Discussion in case any other Users have the same request or would like to track it.  FYI @AnanyaJha
